src/UI/Components/TreeTable/TreeRow/CellWithCopy.tsx

```diff
@@ -32,12 +32,15 @@ export const CellWithCopy: React.FC<Props> = ({ label, value, className }) => {
       {value}
     </Td>
   );
+  const formattedValue = isJson(value)
```

Comment:
Would it not be simpler if you had a function `formatValue` that just returns this string? The `isJson` function can be a part of that. Just an idea. Not required.

Reply:
It was a good idea to extract that as well :+1: